.gitlab/container_build/docker_linux.yml

```diff
@@ -257,6 +258,45 @@ docker_build_cluster_agent_arm64:
   before_script:
     - !reference [.docker_build_s3, before_script]
     - cp -Rvf Dockerfiles/agent/nosys-seccomp Dockerfiles/cluster-agent/
+    - rm ${BUILD_CONTEXT}/datadog-cluster-agent-fips.*
+
+docker_build_cluster_agent_fips_amd64:
+  extends: [.docker_build_job_definition_amd64, .docker_build_s3]
+  rules: !reference [.on_tag_or_a7]
+  needs:
+    - job: cluster_agent_fips-build_amd64
+      artifacts: false
+    - job: cws_instrumentation-build_amd64
+      artifacts: false
+    - job: cws_instrumentation-build_arm64
+      artifacts: false
+  variables:
+    IMAGE: registry.ddbuild.io/ci/datadog-agent/cluster-agent-fips
```

Comment:
Drive-by question: do we need to align the image names between the FIPS agent and FIPS DCA for consistency (I'm not sure what the FIPS image name tag is though)?

Reply:
I think we do, and that aligns with a previous review from Lenaic.

I think we should be good now, but I'll try to push an image to staging to be sure